IPython/utils/traitlets.py

```diff
@@ -1635,14 +1635,14 @@ def validate_elements(self, obj, value):
 class Dict(Instance):
     """An instance of a Python dict."""
 
-    def __init__(self, default_value=None, allow_none=True, **metadata):
+    def __init__(self, default_value={}, allow_none=True, **metadata):
```

Comment:
This PR looks great to me.  I would have merged it quickly for line 1645 below, but I want to make sure the others are okay with the default value change here.  I will wait for one or a few of the others who I think are familiar with traitlets to respond, @ellisonbg @minrk @takluyver and/or @jasongrout ?